ex1b.sh

```diff
@@ -0,0 +1,35 @@
+#1
+ls - S
+
+#1a
+ls -S /etc/ | head -1 
+#Then I found the biggest file that is "services"
+
+#1b
+cat /etc/services
+
+#2
+ls -S -r

```

Comment:
The command is correct, but you can combine the flags -S and -r to simplify the command